capstone/src/main/webapp/text_analyzer.html

```diff
@@ -31,42 +33,24 @@ <h3 class="masthead-brand">Perspective Hub</h3>
         </div>
       </header>
       <h2>Input text and select the Perspective API attribute(s) you would like to be shown</h2>
-      <form>
+      <form onsubmit="submitInput(); return false;">
```

Comment:
nit: It may be worth adding a small comment as to why we `return false` here.

Reply:
Added